draft-privacypass-rate-limit-tokens.md

```diff
@@ -1261,9 +1346,9 @@ b2acb58dcfd23f54fc2084373cd9a78f7c84047c6f64e7464999597267e9b6a0f95bf13f
 44c4ec6abb1c846a16a222116ad94
 request_key: 8a9ffd91064e498f0b14b4b82d65255b2859da8d349e2bb362ca7c8211e
 23fcd5cd01813ef6eb4b994d532c3520e1aa89e
-origin_name_key_id:
+issuer_encap_key_id:
 590ef89ec14ebd7cd0598c7c1e045838e7c484713265c10d923d314fa1ef433f
-encrypted_origin_name: b9d946ee16d9114b6d42524a1fe1e5fae9768c45d4b48ce13
+encrypted_token_request: b9d946ee16d9114b6d42524a1fe1e5fae9768c45d4b48ce13
```

Comment:
This seems not to have been updated. Also we should delete the "blinded_msg" above, and add the issuer response?

Reply:
Yeah, I haven't updated any of the test vectors yet, but will do so before I mark this PR as ready. This particular change is the result of s/encrypted_origin_name/encrypted_token_request.